gitgud/skills/user_messages.py

```diff
@@ -179,7 +179,7 @@ def display_entry(index, human_name, code_name, indent):
             )
 
 
-def basics_status(show_content=True, show_branches=True, working=True, staging=True, file_count=2):
+def basics_status(show_content=True, show_branches=True, working=True, staging=True, file_count=2):  # noqa: E501
```

Comment:
It might make sense to split this method up.

Instead of passing working/staging, would we be able to pass the data itself? We can then have separate methods for working/staging that simply wrap this function, grabbing the appropriate data

Reply:
I like the idea of splitting this function up, but I'm wondering how we'd configure `show_branches`, since that parameter only makes sense for commits and not for the working/staging areas.

Reply:
I think there are several ways. We can set `branch=None` and disregard it.  If someone sets it when they're given working directory/staging content, that's their fault :P You can do it whichever way makes sense.

I think I would have a method `show_stuff(name, directory_content, branch_name=None, file_count=2)` and use that to separate the display function from the other parts of this function. `name` would be `"Working Directory"` etc